app/gen-server/lib/HomeDBManager.ts

```diff
@@ -66,20 +64,25 @@ import {
   Connection,
   DatabaseType,
   EntityManager,
+  ObjectLiteral,
   SelectQueryBuilder,
   WhereExpression
 } from "typeorm";
 import uuidv4 from "uuid/v4";
 import flatten = require('lodash/flatten');
 import pick = require('lodash/pick');
 import defaultsDeep = require('lodash/defaultsDeep');
+import {SUPPORT_EMAIL, UsersManager} from './homedb/UsersManager';
+import {AvailableUsers, GetUserOptions, NonGuestGroup, Resource, UserProfileChange} from './homedb/Interfaces';
+import {normalizeEmail} from 'app/common/emails';
```

Comment:
It seems that imports are organized by  import path aphabetical order, so this line must be brought back @ line 6

Reply:
I reorganized the imports, should be done.